### PR TITLE
Customizable nav colors

### PR DESCRIPTION
**Pull request by kwm4385**
To change the color, simply set the SINGULARITY_NAV_COLOR variable to one of `green`, `red`, `blue`, or `yellow`. 
screen shot 2015-10-27 at 4 03 42 pm
screen shot 2015-10-27 at 4 04 00 pm
screen shot 2015-10-27 at 4 04 18 pm
screen shot 2015-10-27 at 4 05 47 pm

### CODE REVIEW

File: SingularityUI/app/assets/_index.mustache [hunk 1]
```diff
@@ -45,7 +39,7 @@
     <script src="{{{staticRoot}}}/js/app.js"></script>
     <script>require('initialize');</script>
 </head>
-<body>
+<body class="env-color-{{{navColor}}}">
```
Comment (tpetr) [inline SingularityUI/app/assets/_index.mustache]: i'd wrap the class setting in a `{{#if navColor}}` for rigor

**Comment by kwm4385** [comment]
Lets lighten these up.

**Comment by kwm4385** [comment]
Now in pastels just in time for a fabulous spring season:

screen shot 2015-10-29 at 4 45 45 pm
screen shot 2015-10-29 at 4 45 14 pm
screen shot 2015-10-29 at 4 44 53 pm
screen shot 2015-10-29 at 4 45 28 pm

**Comment by kwm4385** [comment]
screen shot 2015-11-11 at 1 23 53 pm
screen shot 2015-11-11 at 1 23 27 pm
screen shot 2015-11-11 at 1 23 08 pm
screen shot 2015-11-11 at 1 23 41 pm

**Pull request merged**